docs/stdlib/type.rst

```diff
@@ -27,6 +27,30 @@ Types
       - :eql:op-desc:`introspect`
 
 
+Finding an object's type
+------------------------
+
+You can find the type of an object via that object's ``__type__`` link, which
```

Comment:
I nearly just copy/pasted wholesale [our documentation of this in the `select` docs](https://www.edgedb.com/docs/edgeql/select#accessing-types-in-polymorphic-queries), but I pulled back thinking this might not be the place for it. This page feels more like a reference for type operators, and including this feels a bit like an intrusion into that. I'm open to dissent though. I could include more here, or I could link over to the expanded information about it in the `select` docs. What do you think?

Reply:
Yeah, not a hard blocker which is why I approved anyway. While there is a ton of good info in that linked documentation section, I think the only thing I'm suggesting here is to explicit call out the lack of magic since the bare syntax alone makes you wonder how and why this `__type__` link exists on "an object".

Definitely not a blocker, and if this makes it hard to have a nice terse flow here, I'm fine with what you've already provided!

Reply:
I like the suggestion. Added a short bit about it that I don't think makes this overly long. Thank you for the review and the suggestion!

Reply:
Although, for some reason, it doesn't show as approved. 🤔 Can you try it again?